### ISSUE TITLE
BrushSelector does not update when selection changed programmatically

### ISSUE THREAD

**Issue by astrofrog** [bqplot/bqplot#1067]
**Describe the bug**

If ``selected_x`` and/or ``selected_y`` are changed on a brush selector, the visible selection rectangle does not update immediately. Instead, the browser window needs to be resized or refreshed.

**To Reproduce**

Create a figure with a brush selector:

```python
from bqplot import pyplot as plt
from bqplot.interacts import BrushSelector
fig = plt.figure()
plt.plot([1, 2, 3], [2, 4, 3])
plt.show()
interact = BrushSelector(x_scale=fig.axes[0].scale, y_scale=fig.axes[1].scale)
fig.interaction = interact
```

Then click and drag to draw a shape. Once this is done, change ``selected_x`` and ``selected_y``:

```python
interact.selected_x = [1.4, 2.4]
interact.selected_y = [2.4, 4]
```

The rectangle does not update unless the window is resized or refreshed.

**Expected behavior**

The brush selector should update immediately

**Screenshots**

![Peek 2020-03-25 14-06](https://user-images.githubusercontent.com/314716/77551829-b21d9400-6eaa-11ea-9998-f76a7a9cd58c.gif)

**Environment (please complete the following information):**
 - Operating System and Version: Ubuntu 20.04 (pre-release)
 - Browser: Firefox

**Additional context**
It looks like this is still an issue with the latest developer version of bqplot

**Comment by maartenbreddels** [comment]
Also, I expect:
```
fig.interaction = None
fig.interaction = interact
```
To draw the BrushSelector.
The workaround, to force it to draw is to modify fig_margin:
```
fig.fig_margin = {'top': 60, 'bottom': 60, 'left': 60, 'right': 61}
```

**Comment by SylvainCorlay** [comment]
Fixed in #1069 !
